sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -418,4 +431,19 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       assert(sql("SHOW PARTITIONS part_datasrc").count() == 3)
     }
   }
+
+  /**
+   * Run a function with a copy of the input file. Use this for tests that use "LOAD DATA"
+   * (instead of "LOAD DATA LOCAL") since, according to Hive's semantics, files are moved
```

Comment:
Ah, the tests need to be updated because now `loadTable` is being called with "isSrcLocal = false". That makes the source file be moved instead of copied, and that makes subsequent unit tests fail. (That's the cause of the initial test failures in this PR.)